ros/src/waypoint_loader/waypoint_loader.py

```diff
@@ -66,7 +66,7 @@ def decelerate(self, waypoints):
         for wp in waypoints[:-1][::-1]:
             dist = self.distance(wp.pose.pose.position,
                                  last.pose.pose.position)
-            vel = math.sqrt(2 * MAX_DECEL * dist) * 3.6
+            vel = math.sqrt(2 * MAX_DECEL * dist)
```

Comment:
??

Reply:
That factor was transforming `vel` from m/s to km/h. It's better to have everything in m/s.

Reply:
I agree. lets add comment or helper function